egui_web/src/http.rs

```diff
@@ -24,6 +24,10 @@ async fn fetch_jsvalue(request: &Request) -> Result<Response, JsValue> {
     opts.method(method);
     opts.mode(web_sys::RequestMode::Cors);
 
+    if method == &"POST".to_string() {
```

Comment:
`method == "POST"` should work fine

Reply:
I have tested manually with my personal development http server. I use kore from kore.io. 
By default the Content-Type is plain/text. If we would send binary data then we need to change it, but since web_sys only has the body referenced by JsValue (and I am new to Rust - less than a week) I believe that we need only to send it if we want to change. 

I the GLIUM version since it uses ureq it has the separate functions to send binary, text, form or json.

Reply:
> I have tested manually with my personal development http server.

Great! Web or glium version or both?

> I am new to Rust - less than a week

Wow! Good work on this PR, and welcome to the wonderful world of Rust!